flang/include/flang/Semantics/openmp-directive-sets.h

```diff
@@ -24,68 +24,80 @@ namespace llvm::omp {
 // - all<Directive>Set: All standalone or combined uses of the directive.
 
 static const OmpDirectiveSet topParallelSet{
-    Directive::OMPD_parallel,
-    Directive::OMPD_parallel_do,
     Directive::OMPD_parallel_do_simd,
+    Directive::OMPD_parallel_do,
+    Directive::OMPD_parallel_masked_taskloop_simd,
+    Directive::OMPD_parallel_masked_taskloop,
+    Directive::OMPD_parallel_master_taskloop_simd,
+    Directive::OMPD_parallel_master_taskloop,
     Directive::OMPD_parallel_sections,
     Directive::OMPD_parallel_workshare,
+    Directive::OMPD_parallel,
```

Comment:
How are you sorting these directives?  When I run "sort" (the Unix command), I get the OMPD_parallel first in the output.

Reply:
I used VSCode's sort lines command, but I'm not quite sure what happened. It might have been related to the commas, I think. I tried again without them and it looks like the ordering made more sense. Just created an upstream PR with these changes.